src/main/java/net/minecraftforge/client/model/pipeline/BlockInfo.java

```diff
@@ -28,7 +28,7 @@
 
 public class BlockInfo
 {
-    private static final EnumFacing[] SIDES = EnumFacing.values();
+    private static final EnumFacing[] SIDES = EnumFacing.VALUES;
```

Comment:
I do not agree with this change - it adds a dependency on mutable state for no gain